### ISSUE TITLE
Combine will not start

### ISSUE THREAD

**Issue by sincosam** [Courseplay/courseplay#3042]
Hello I am having a problem with getting any combine to start working. The combine will extend it offload pipe and freeze in place will not move, I have to hire an ingame worker to correct the issue. I have tried many different combines and diffrent maps. I am using v6.01.00015 and have uploaded a log file 
https://pastebin.com/fUZVrw2b 

Please let me know if you need anything else

Thanks,
Sam

**Comment by pvajko** [comment]
Yes, the log with debug channels 12 and 14 and a clear description of what exactly were your doing.

**Comment by Dajoor** [comment]
I have the same problem. ALL stock and mod combines on ALL stock and mod maps. It matters none how I set the course, it will go to the first point, activate the header, then extend the output pipe, fold the combine,  and be stuck. Then I must reset the combine. Below the output of the debug channels:
 
2019-01-31 22:21 :06 [dbg14 lp12175] T560 (#1): There seems to be no unload/refill course
2019-01-31 22:21 :06 [dbg14 lp12175] T560 (#1): change to fieldwork
2019-01-31 22:21 :06 [dbg14 lp12175] T560 (#1): Starting work: turn on and lower implements.
2019-01-31 22:21 :06 [dbg14 lp12176] T560 (#1): onWaypointChange 3
2019-01-31 22:21 :06 [dbg14 lp12176] T560 (#1): onWaypointPassed 1
2019-01-31 22:21 :06 [dbg14 lp12200] T560 (#1): Chopper waiting for trailer, fill level 9.975000
2019-01-31 22:21 :06 [dbg14 lp12200] T560 (#1): Speed = 0.0, gx=0.0 gz=5.0 l=5.0 ax=0.0 az=5.0
2019-01-31 22:21 :07 [dbg14 lp12239] T560 (#1): all tools ready, start working
2019-01-31 22:21 :09 [dbg14 lp12300] T560 (#1): Chopper waiting for trailer, fill level 9.975000
2019-01-31 22:21 :09 [dbg14 lp12300] T560 (#1): Fill levels: OAT: 0.0
2019-01-31 22:21 :09 [dbg14 lp12300] T560 (#1): Fill levels: DIESEL: 99.7
2019-01-31 22:21 :09 [dbg14 lp12300] T560 (#1): Fill levels: DEF: 99.8
2019-01-31 22:21 :09 [dbg14 lp12300] T560 (#1): Speed = 0.0, gx=0.0 gz=5.0 l=5.0 ax=0.0 az=5.0
2019-01-31 22:21 :12 [dbg14 lp12400] T560 (#1): Chopper waiting for trailer, fill level 9.975000
2019-01-31 22:21 :12 [dbg14 lp12400] T560 (#1): Fill levels: OAT: 0.0
2019-01-31 22:21 :12 [dbg14 lp12400] T560 (#1): Fill levels: DIESEL: 99.7
2019-01-31 22:21 :12 [dbg14 lp12400] T560 (#1): Fill levels: DEF: 99.8
2019-01-31 22:21 :12 [dbg14 lp12400] T560 (#1): Speed = 0.0, gx=0.0 gz=5.0 l=5.0 ax=0.0 az=5.0
2019-01-31 22:21 :16 [dbg14 lp12500] T560 (#1): Chopper waiting for trailer, fill level 9.975000

5 lines above repeats infinitely.

**Comment by pvajko** [comment]
I need your full log. This looks like CP thinks your combine is a chopper but if no combines worked at all with CP then probably others would have complained about it. Something else must be going on there too.

**Comment by Dajoor** [comment]
Thanks for pointing that chopper thing out. I did not see that, actual reading FTW. My issue is resolved - it was caused by another mod.

Perhaps a suggestion: 
toolManager.lua : 218
```
function courseplay:isChopper(workTool)
	return workTool.cp.hasSpecializationCombine and workTool.startThreshing ~= nil and workTool.cp.capacity == 0 or courseplay:isSpecialChopper(workTool);
end;
```
Maybe checking capacity == 0 is not the best way?

And, while searching for the problem, I also noticed this:
drive.lua : 1830
`or fillUnit.capacity > 99999 `

This might cause issues with some modified vehicles/equipment.

Many Thanks
